scene/gui/color_picker.cpp

```diff
@@ -2861,8 +2164,11 @@ ColorPicker::ColorPicker() {
 }
 
 ColorPicker::~ColorPicker() {
-	for (int i = 0; i < modes.size(); i++) {
-		delete modes[i];
+	for (ColorMode *mode : modes) {
+		delete mode;
+	}
+	for (ColorPickerShape *shape : shapes) {
+		delete shape;
 	}
```

Comment:
I wonder why `delete` is used here. Don't we use `memdelete()` typically?

Reply:
Well, the ColorMode code was a GSoC project and this probably slipped past review. I just left it as is, but makes sense to change it.